docs/products/flink/getting-started.rst

```diff
@@ -1,12 +1,74 @@
 Getting started
 ===============
 
-Aiven for Flink is available in the `Aiven console <https://console.aiven.io>`_.
+The first steps in using Aiven for Apache Flink are to create a service and a Flink job. You can do this in the Aiven web console or with the Aiven CLI.
 
-Choose the Apache Flink version, your cloud provider and location to deploy to, then choose which plan to use.
+In addition to the Flink service, you also need a service that you can use as a data source. This article outlines an example of using an Apache Kafka service as the data source.
 
-Finally, give the service a name and then select "Create Service", and your new Flink service will start building. While it does that, you can already visit the service overview page to see the details of the service.
 
-.. image:: /images/products/flink/console-overview.png
+Create an Apache Flink service in the Aiven web console
+-------------------------------------------------------
 
 
+1. Log in to the `Aiven web console <https://console.aiven.io/>`_.
+
+2. On the *Services* page, click **Create a new service**.
+
+   This opens a new page with the available service options.
+
+
+3. Select the main properties for your service:
+
+   a. Select **Flink** as the service type.
+
+   b. Select the cloud provider and region that you want to run your service on.
+
+      .. note:: 
+	      The pricing for the same service may vary between different providers and regions. The service summary on the right side of the console shows you the pricing for your selected options.
+
+   c. Select a service plan.
+
+      This defines the number of servers and what kind of memory, CPU, and disk resources are allocated to your service.
+
+   d. Enter a name for your service.
+
+      A random name is provided by default, but you can enter a more recognizable name to distinguish it from other services.
+
+
+4. Click **Create Service** under the summary on the right side of the console.
+
+   This brings you back to the **Services** view. Your new service is listed with a status indicator to show that it is being created.
+
+5. Click the service name.
+
+   The *Overview* page for the service opens.
```

Comment:
should *Overview* be **Overview**?

Reply:
Similar to the earlier comment, I'd prefer to stick with italics for non-interactive UI elements